tests/python/pytorch/graphbolt/test_feature_store.py

```diff
@@ -1,13 +1,28 @@
+import numpy as np
 import pytest
 import torch
 from dgl import graphbolt as gb
 
 
-def test_in_memory_feature_store():
+def to_on_disk_tensor(name, t):
+    path = f"/tmp/{name}.npy"
+    t = t.numpy()
+    np.save(path, t)
+    t = torch.as_tensor(np.load(path, mmap_mode="r+"))
```

Comment:
no RAM is consumed at all until `t` is accessed by indices?

Reply:
Yes. And only the returned tensor of `read` consumes memory.